File: runtime/CSharp/Antlr4.Runtime/Atn/DecisionState.cs
// Copyright (c) Terence Parr, Sam Harwell. All Rights Reserved.
// Licensed under the BSD License. See LICENSE.txt in the project root for license information.

using Antlr4.Runtime.Sharpen;

namespace Antlr4.Runtime.Atn
{
    public abstract class DecisionState : ATNState
    {
        public int decision = -1;

        public bool nonGreedy;

        public bool sll;
    }
}

File: runtime/Java/src/org/antlr/v4/runtime/atn/DecisionState.java
/*
 * Copyright (c) 2012 The ANTLR Project. All rights reserved.
 * Use of this file is governed by the BSD-3-Clause license that
 * can be found in the LICENSE.txt file in the project root.
 */

package org.antlr.v4.runtime.atn;

public abstract class DecisionState extends ATNState {
	public int decision = -1;
	public boolean nonGreedy;
	public boolean sll;
}
